### PR TITLE
Update README, .travis.yml for Melodic

### PR DESCRIPTION
**Pull request by xabxx**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by xabxx** [comment]
1. Verified `ros-melodic-cloudwatch-logger` in ROS melodic
2. Passed Travis https://travis-ci.org/aws-robotics/cloudwatchmetrics-ros1/builds/520487495?utm_source=github_status&utm_medium=notification

File: README.md [hunk 1]
```diff
@@ -23,8 +23,7 @@ The source code is released under an [Apache 2.0].
 
 * Travis CI: [![Build Status](https://travis-ci.org/aws-robotics/cloudwatchmetrics-ros1.svg?branch=master)](https://travis-ci.org/aws-robotics/cloudwatchmetrics-ros1)
  * ROS build farm:
-   * v1.0.0:
-     * ROS Kinetic @ u16.04 Xenial [![Build Status](http://build.ros.org/job/Kbin_uX64__cloudwatch_metrics_collector__ubuntu_xenial_amd64__binary/badge/icon)](http://build.ros.org/job/Kbin_uX64__cloudwatch_metrics_collector__ubuntu_xenial_amd64__binary)
+   * ROS Kinetic @ u16.04 Xenial [![Build Status](http://build.ros.org/job/Kbin_uX64__cloudwatch_metrics_collector__ubuntu_xenial_amd64__binary/badge/icon)](http://build.ros.org/job/Kbin_uX64__cloudwatch_metrics_collector__ubuntu_xenial_amd64__binary)
```
Comment (AAlon) [inline README.md]: Missing Melodic build status?

Comment (xabxx) [reply README.md]: added melodic build status.

Comment (xabxx) [reply README.md]: added melodic build status

**Pull request merged**